CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 <a name="0.1.0"></a>
-## 0.1.0 (2021/08/08)
```

Comment:
This shouldn't be changed. Check if your line endings is in Linux style (LF) instead of Windows style (CRLF)
https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings